server/einterfaces/metrics.go

```diff
@@ -102,4 +102,7 @@ type MetricsInterface interface {
 	IncrementNotificationErrorCounter(notificationType model.NotificationType, errorReason model.NotificationReason)
 	IncrementNotificationNotSentCounter(notificationType model.NotificationType, notSentReason model.NotificationReason)
 	IncrementNotificationUnsupportedCounter(notificationType model.NotificationType, notSentReason model.NotificationReason)
+
+	IncrementClientChannelVisited(platform, agent string, inc float64)
+	ObserveClientChannelLoadTime(platform, agent string, elapsed float64)
```

Comment:
I guess this follows the existing pattern for these other metrics, but I would've expected that there was a more generic API for this to avoid having to make changes to the enterprise repo every time we want to add a new metric. As-is, there's 3 places we need to specify these metrics between the web app, server, and enterprise code now.

That said, that's definitely not a blocker for this, especially since it matches the existing metrics

Reply:
Actually, I forgot that the enterprise code for metrics is in this repo, so that's a non-inssue since we don't need to make multiple PRs anyway

Reply:
Yeah, that's why I wanted to strongly type the metrics. (to ensure server doesn't receive unexpected metrics). We can think about the alternative. But it can prone to cases where a client *can* create infinite number of metrics.

Reply:
Oh yeah, that makes sense. It's too bad that Go's type system doesn't support us making the metric name into an enum, and even if we could trust that people aren't calling the API maliciously, TS only confirms its types at compile time